src/components/ui/Avatar.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from "react";
+
+export default class Avatar extends Component {
+    constructor(props) {
+        super(props);
+    }
+    render() {
+        return (
+            <div className="avatar">
+                <div className="avatarImg">
+                    <img src={this.props.image} alt="#" />
+                </div>
```

Comment:
Would you recommend uploading a picture? It would be nice, but what should we do if we didn't?

Reply:
It looked very nice, so i took it;)
We could make the initial (as google does for example)